generators/common/templates/README.md.ejs

```diff
@@ -280,7 +280,7 @@ Sonar is used to analyse code quality. You can start a local Sonar server (acces
 docker-compose -f src/main/docker/sonar.yml up -d
 ```
 
-You can both run sonar with maven or gradle but also with a sonar-scanner.
+You can both run Sonar with maven or gradle but also with a sonar-scanner.
```

Comment:
I would rephrase the sonar scanner sentence. It is a bit annoying as when using --skip-client you are using neither maven or gradle so it makes no sense to mention them ! So I would write it like that:

`You can run a Sonar analysis with using the [sonar-scanner](https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner)<% if (buildTool) { %> or by using the <% if (buildTool === 'maven') { %>maven<% } %><% if (buildTool === 'gradle') { %>gradle<% } %> plugin<% } %> .`

Reply:
OFC it's better and I alslo remove the outdated sonar-scanner part below